project/Dependencies.scala

```diff
@@ -16,7 +16,7 @@ object Dependencies {
     // We don't force Akka updates because downstream projects can upgrade
     // themselves. For more information see
     // https://doc.akka.io//docs/akka/current/project/downstream-upgrade-strategy.html
-    val akka = "2.6.9"
+    val akka = "2.6.13"
```

Comment:
Did you have a particular reason to want to update to Akka 2.6.13?

As mentioned in the comment above, we typically don't update Akka because downstream projects are free to upgrade Akka themselves anyway, so this gives them more flexibility.

Reply:
No reason really, can revert